src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -40,6 +40,7 @@ struct miopen_convolution
     shared<convolution_descriptor> cd = nullptr;
     miopenConvFwdAlgorithm_t algo{};
     uint64_t solution_id = 0;
+    value::binary solution_object{};
```

Comment:
This needs to store the `miopenSolution_t` object(this actually needs to be a managed pointer).

Reply:
Didnt' find a way to create a managed_ptr since, `loadSolution` requires  pointer to pointer.

Reply:
Added `shared_ptr` for miopenSolution.